src/rich-text-editor.component.js

```diff
@@ -85,13 +87,24 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     }
   }, [])
 
+  useEffect(() => {
+    if (props.placeholder) {
+      const styleElement = document.createElement('style')
+      styleElement.textContent = `.bandicoot-id-${bandicootId.current}:empty:not(:focus):before { content: attr(data-placeholder); color: ${props.placeholderColor}; }`
+      document.head.appendChild(styleElement)
+
+      return () => styleElement.parentNode.removeChild(styleElement)
+    }
+  },[props.placeholder, bandicootId.current])
```

Comment:
Since `props.placeholderColor` is used inside of this effect, it should be added to the conditions array for the effect. See https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect for more info